packages/notifi-react-card/lib/context/NotifiContext.tsx

```diff
@@ -52,6 +52,14 @@ export type AcalaParams = Readonly<{
   signMessage: AcalaSignMessageFunction;
 }>;
 
+export type OptimismParams = Readonly<{
+  walletBlockchain: 'OPTIMISM';
+  accountAddress: string;
```

Comment:
maybe i'm missing sometning, but other EVM params don't require accountAddress -- I think we should copy/paste an EVM param payload instead.

Better: can you take a stab and DRYing up the EVM params? it'll probably work if we do something like
```
export type EvmParams = Readonly<{
  walletBlockchain: 'ETHEREUM' | 'POLYGON' | ... | 'OPTIMISM';
  ...
}>
```

I don't know what ramifications that would have on the rest of the codebase though, so if it's too complicated just abort

Reply:
can give it a whirl - whats the appropriate approach to testing the sdk?

Reply:
it's been fairly manual. 
i think if you can logIn with the wallet via the notifi-react-example that would be good

Reply:
Creating backlog ticket -- will be an issue in the future. Have to clean up referencing for Wallet Params as Notifi-Core seems like the source of truth.